Tests/IntegrationTests/Components Tests/PayTo/PayToComponentTests.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) @testable import Adyen
+@testable import AdyenComponents
+import XCTest
+
+class PayToComponentTests: XCTestCase {
```

Comment:
Could you add tests to `PaymentMethodTests` as well?

Reply:
e1d30269b308ad40e52cd9c0f869fd92039d3c43